src/witan/send/adroddiad/valid_states.clj

```diff
@@ -20,22 +20,24 @@
     (ay/key-stage-3 y) (apply max ay/key-stage-3)
     (ay/key-stage-4 y) (apply max ay/key-stage-4)
     (ay/key-stage-5 y) (apply max ay/key-stage-5)
-    (ay/ncy-15+ y) (apply max ay/ncy-15+)))
+    (ay/ncy-15+ y) (apply max ay/ncy-15+)
+    (< (apply max ay/ncy-15+) y) (apply max ay/ncy-15+)))
 
 (defn candidate-valid-states [census-data]
   (let [needs (->> (into (sorted-set) (:need census-data))
                    (interpose \,)
                    (apply str))
         settings (->> (into (sorted-set) (:setting census-data))
                       (interpose \,)
-                      (apply str))]
+                      (apply str))
+        ncys (into (sorted-set) (:academic-year census-data))]
     (-> census-data
         (tc/select-columns [:setting :academic-year])
         (tc/group-by (fn [row] (-> (:setting row)
                                    (clojure.string/split #"_")
                                    first)))
-        (tc/aggregate {:min-academic-year #(keystage-min-ay (reduce min (:academic-year %)))
-                       :max-academic-year #(keystage-max-ay (reduce max (:academic-year %)))})
+        (tc/aggregate {:min-academic-year #(max (apply min ncys) (keystage-min-ay (reduce min (:academic-year %))))
```

Comment:
I'm sort of OK with this one as there are more issues in the early years data where I don't want to create a joiner prior or others for things we haven't seen. I do tend to edit this afterwards as I see this function more as creating a starting template than the final output.